extra/1-currency-conversion.js

```diff
@@ -5,7 +5,9 @@
   Write a function that converts a price to USD (exchange rate is 1.4 $ to £)
 */
 
-function convertToUSD() {}
+function convertToUSD(poundToUsd) {
+  return poundToUsd * 1.4;
+}
 
```

Comment:
Well done on solving an extra assignment!!
The parameter poundToUsd sounds as if it's a function.
For further improvement, you may want to change it into pound, amountInPound, or something more semantic.